airflow/utils/log/log_reader.py

```diff
@@ -100,6 +101,11 @@ def is_supported(self):
 
         return hasattr(self.log_handler, 'read')
 
+    @property
+    def is_external(self):
+        """Check if the logging handler is external (e.g. Elasticsearch, Stackdriver, etc)."""
```

Comment:
```suggestion
    def supports_external_link(self):
        """Check if the logging handler is external links (e.g. Elasticsearch, Stackdriver, etc)."""
```
This name is because the word "external" suggests that the class is not part of the Airflow core, but we want to check here whether external links are supported.  When we introduce it, we need to improve the "A" property so that it has a similar naming convention - `support_read`

Reply:
Makes sense.